pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -950,9 +960,25 @@ func ValidateVirtualMachineInstanceMandatoryFields(field *k8sfield.Path, spec *v
 	return causes
 }
 
-func ValidateVirtualMachineInstanceMetadata(field *k8sfield.Path, metadata *metav1.ObjectMeta, config *virtconfig.ClusterConfig) []metav1.StatusCause {
+func ValidateVirtualMachineInstanceMetadata(field *k8sfield.Path, metadata *metav1.ObjectMeta, config *virtconfig.ClusterConfig, accountName string) []metav1.StatusCause {
+
 	var causes []metav1.StatusCause
 	annotations := metadata.Annotations
+	labels := metadata.Labels
```

Comment:
I would write this as follow: 
1. Check if the user is allowed to create restricted kubevirt.io labels 
2. If not, check if did...

Reply:
@xpivarc Done